tgui/packages/tgui/interfaces/SpecMenu.js

```diff
@@ -0,0 +1,99 @@
+        <p><b>Blood eruption</b>: unlocked at 600 blood, allows you to manipulate all nearby blood splatters into spikes that impale anyone stood ontop of them.</p>
+        <p><b>Full power</b>
+          <Divider />
+          <b>The blood bringers rite</b>: when toggled you will rapidly drain the blood of everyone who is nearby and use it to heal yourself and remove stamina damage.
+        </p>
+        <Button
+          content="Hemomancer"
+          onClick={() => act('hemomancer')}
+        />
+      </Section>
+    </Flex.Item>
+  );
+};
+
+const UmbrMenu = (props, context) => {
+  const { act, data } = useBackend(context);
+  const {
+    subclasses,
+  } = data;
+  return (
+    <Flex.Item grow={1} basis="33%">
+      <Section title="Umbrae">
+        <p>Focuses on darkness, stealth ambushing and mobility.</p>
+        <p><b>Cloak of darkness</b>: unlocked at 150 blood, when toggled, allows you to become nearly invisible and move rapidly when in dark regions.</p>
+        <p><b>Shadow snare</b>: unlocked at 250 blood, allows you to summon a trap that when crossed blinds and ensares the victim. This trap is hard to see, but withers in the light.</p>
+        <p><b>Dark passage</b>: unlocked at 400 blood, allows you to target a turf on screen, you will then teleport to that turf.</p>
+        <p><b>Extinguish</b>: unlocked at 600 blood, allows you to snuff out nearby electronic light sources and glowshrooms.</p>
+        <p><b>Full power</b>
+          <Divider />
+          <b>Eternal darkness</b>: when toggled, you consume yourself in unholy darkness, only the strongest of lights will be able to see through it. It will also cause nearby creatures to freeze.
+        </p>
+        <p>In addition, you also gain permament X-ray vision.</p>
+        <Button
+          content="Umbrae"
+          onClick={() => act('umbrae')}
+        />
+      </Section>
+    </Flex.Item>
+  );
+};
+
+const GarMenu = (props, context) => {
+  const { act, data } = useBackend(context);
+  const {
+    subclasses,
+  } = data;
+  return (
+    <Flex.Item grow={1} basis="33%">
+      <Section title="Gargantua">
+        <p>Focuses on tenacity and melee damage.</p>
+        <p>Rejuvenate, will heal you at an increased rate based on how much damage you have taken.</p>
+        <p><b>Blood swell</b>: unlocked at 150 blood, increases your resistance to physical damage, stuns and stamina.</p>
+        <p><b>Blood rush</b>: unlocked at 250 blood, gives you a short speed boost when cast.</p>
+        <p><b>Blood swell II</b>: unlocked at 400 blood, increases all melee damage by 10.</p>
+        <p><b>Overwhelming force</b>: unlocked at 600 blood, when toggled, if you bump into a door that you dont have access to, it will force it open.</p>
+        <p><b>Full Power</b>
+          <Divider />
+          <b>Charge</b>: you gain the ability to charge at a target. Destroying and knocking back pretty much anything you collide with.
+        </p>
```

Comment:
```suggestion
        <h3>Focuses on tenacity and melee damage.</h3>
        <Divider />
        <p><b>Rejuvenate</b>: Will heal you at an increased rate based on how much damage you have taken.</p>
        <p><b>Blood swell</b>: Unlocked at 150 blood, increases your resistance to physical damage, stuns and stamina.</p>
        <p><b>Blood rush</b>: Unlocked at 250 blood, gives you a short speed boost when cast.</p>
        <p><b>Blood swell II</b>: Unlocked at 400 blood, increases all melee damage by 10.</p>
        <p><b>Overwhelming force</b>: Unlocked at 600 blood, when toggled, if you bump into a door that you dont have access to, it will force it open.</p>
        <p><b>Full Power</b>
          <Divider />
          <b>Charge</b>: You gain the ability to charge at a target. Destroying and knocking back pretty much anything you collide with.
        </p>
```
(Also, I think the rejuvenate one was meant to be bolded like the others?)